gem2rpm/smart_proxy_plugin.spec.erb

```diff
@@ -6,15 +6,19 @@ SCL_PREFIXES = JSON.load(File.read(File.join(`git rev-parse --show-toplevel`.str
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
 
+%{!?_root_datadir:%global _root_datadir %{_datadir}}
+%{!?_root_localstatedir:%global _root_localstatedir %{_localstatedir}}
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+
 %global gem_name <%= spec.name %>
 %global plugin_name <%= spec.name.sub(/\Asmart_proxy_/, '') %>
 
 %global foreman_proxy_min_version 1.24
-%{!?_root_datadir:%global _root_datadir %{_datadir}}
 %global foreman_proxy_dir %{_root_datadir}/foreman-proxy
+%global foreman_proxy_statedir %{_root_localstatedir}/foreman-proxy
 %global foreman_proxy_bundlerd_dir %{foreman_proxy_dir}/bundler.d
 %global foreman_proxy_settingsd_dir %{_root_sysconfdir}/foreman-proxy/settings.d
-%global smart_proxy_dynflow_bundlerd_dir %{_datadir}/smart_proxy_dynflow_core/bundler.d
+%global smart_proxy_dynflow_bundlerd_dir %{_root_datadir}/smart_proxy_dynflow_core/bundler.d
```

Comment:
I think this isn't actually in the root but *always* in the SCL when the platform has SCL support (essentially that's EL7 for our purposes). Regardless of whether this package itself is built in SCL.

Reply:
I don't follow exactly. But I can tell you I had to do define it this way to ensure everything worked in both environments.